<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservationViewer</class>
 <widget class="QDialog" name="ReservationViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>572</width>
    <height>524</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>572</width>
    <height>524</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reservation Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Icons/clock104.png</normaloff>:/Icons/clock104.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reservation ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>End :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>For :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Created By :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Location :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="notesField">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes on reservation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Is repeated :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="invTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>551</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="resIdLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="endLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="forLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="createdByLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="repeatedLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
